### PR TITLE
v9.7 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/deploy/mattermost-changelog.md [hunk 1]
```diff
@@ -15,6 +15,80 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Fixed an issue where the Desktop App login flow would erroneously show the landing page for first time users.
+ - Fixed an issue where a right-hand side card was not reloaded when the card body was updated.
+ - Fixed an issue where ``en-AU`` language selection was not allowed.
+ - Fixed an issue with the position of text in the default profile picture.
+ - Fixed an issue with the group search to parse the display name.
+ - Fixed an issue where items with longer text did not widen the user guide dropdown to its max-width.
+ - Fixed an issue where the configuration could not be updated from the **System Console** in cloud environments.
+
+### config.json
+ - Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
```
Comment (wiggin77) [inline source/deploy/mattermost-changelog.md:69]: >  Below is a list of the additions and their default values on install

Where is this list?

Comment (amyblais) [reply source/deploy/mattermost-changelog.md:69]: Below it on line 71 https://github.com/mattermost/docs/pull/6985/files#diff-0c47e93e01b6bf186c28619073df3eb12f83259e7181ea91fffa1c48575d2296R71.

**Review by nickmisasi — APPROVED** [review]

**Review by cpoile — APPROVED** [review]

**Review by streamer45 — APPROVED** [review]

**Review by agnivade — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 2]
```diff
@@ -7,10 +7,86 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Upgraded ``@mattermost/client`` and ``@mattermost/types`` to support TypeScript v5.x.
+ - The first emoji is now auto-selected in the emoji picker.
+ - Added markdown support for batched email notifications.
+ - Users’ timezone is now used in batched email notifications.
+ - Removed a conflicting class (``help-text``) from the interactive dialog field description to resolve the black text color in the dark theme.
+ - Updated the user interface of **Team Settings** modal.
+ - Promoted Simplified Chinese to Beta, and downgraded Hungarian and Spanish languages to Beta.
+ - Improved the opening animation of the user settings modal.
+
+#### Administration
```
Comment (marianunez) [inline source/deploy/mattermost-changelog.md:38]: @amyblais can we include the change for the safety limits that we introduced with https://github.com/mattermost/mattermost/pull/26511 cc// @jasonblais 

```suggestion
#### Administration

- Enforced safety limit in compiled Team Edition and Enterprise Edition deployments when enterprise scale and access control automation features are unavailable and count of users who are registered and not deactivated exceeds 11,000. ERROR_SAFE_LIMITS_EXCEEDED.
```

File: source/deploy/mattermost-changelog.md [hunk 3]
```diff
@@ -7,10 +7,87 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Enforced safety limit in compiled Team Edition and Enterprise Edition deployments when enterprise scale and access control automation features are unavailable and count of users who are registered and not deactivated exceeds 11,000. ERROR_SAFE_LIMITS_EXCEEDED.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:39]:
```suggestion
 - Enforced safety limit in compiled Team Edition and Enterprise Edition deployments when enterprise scale and access control automation features are unavailable, and when the count of users who are registered, but not deactivated, exceeds 11,000. ERROR_SAFE_LIMITS_EXCEEDED.
```

File: source/deploy/mattermost-changelog.md [hunk 4]
```diff
@@ -7,10 +7,87 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Dropped pre-packaged plugins for unsupported OS and architectures.
+ - Implemented a new **Export Settings** page in the **System Console** to allow Cloud administrators to customize their dedicated export S3 buckets.
+ - LDAP job details are no longer shown until the job runs.
+ - Added more logging to the ``NotificationsLog``.
+ - A message is now logged if a user tries to login using a wrong password.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:44]:
```suggestion
 - A message is now logged when a user tries to log in using an incorrect password.
```

File: source/deploy/mattermost-changelog.md [hunk 5]
```diff
@@ -7,10 +7,87 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+### Improvements
+
+#### User Interface (UI)
+ - Added Mattermost [AI plugin](https://github.com/mattermost/mattermost-plugin-ai) to pre-packaged plugins.
+ - Pre-packaged Calls version [v0.25.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.25.1).
+ - Pre-packaged Playbooks version [v1.39.2](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v1.39.2).
+ - Pre-packaged GitHub plugin version [v2.2.0](https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.2.0).
+ - Upgraded ``@mattermost/client`` and ``@mattermost/types`` to support TypeScript v5.x.
+ - The first emoji is now auto-selected in the emoji picker.
+ - Added markdown support for batched email notifications.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:31]:
```suggestion
 - Added Markdown support for batched email notifications.
```

**Review by cwarnermm — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 6]
```diff
@@ -7,10 +7,87 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Users’ timezone is now used in batched email notifications.
+ - Removed a conflicting class (``help-text``) from the interactive dialog field description to resolve the black text color in the dark theme.
+ - Updated the user interface of **Team Settings** modal.
+ - Promoted Simplified Chinese to Beta, and downgraded Hungarian and Spanish languages to Beta.
+ - Improved the opening animation of the user settings modal.
+
+#### Administration
+ - Enforced safety limit in compiled Team Edition and Enterprise Edition deployments when enterprise scale and access control automation features are unavailable, and when the count of users who are registered, but not deactivated, exceeds 11,000. ERROR_SAFE_LIMITS_EXCEEDED.
+ - Dropped pre-packaged plugins for unsupported OS and architectures.
+ - Implemented a new **Export Settings** page in the **System Console** to allow Cloud administrators to customize their dedicated export S3 buckets.
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/deploy/mattermost-changelog.md:41]: @cwarnermm Here it is: https://github.com/mattermost/mattermost/pull/26034. I had added the Docs/Needed label, but you had recently removed the label, so maybe you had already reviewed that PR.

[the remark this answers is not in the document]
Comment (cwarnermm) [reply source/deploy/mattermost-changelog.md:41]: aha! Yes! Now I remember -- this one's behind a feature flag, so no need for doc updates yet. Thanks, @amyblais! :)

File: source/deploy/mattermost-changelog.md [hunk 7]
```diff
@@ -11,6 +11,87 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Added Mattermost [AI plugin](https://github.com/mattermost/mattermost-plugin-ai) to pre-packaged plugins.
+ - Pre-packaged Calls version [v0.25.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.25.1).
+ - Pre-packaged Playbooks version [v1.39.2](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v1.39.2).
+ - Pre-packaged GitHub plugin version [v2.2.0](https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.2.0).
+ - Upgraded ``@mattermost/client`` and ``@mattermost/types`` to support TypeScript v5.x.
```
Comment (jespino) [inline source/deploy/mattermost-changelog.md:33]: This is not UI related

**Pull request merged**